### ISSUE TITLE
Randomly loosing everything and rescraping

### ISSUE THREAD

**Issue by okan35** [nova-video-player/aos-AVP#449]
So sometimes app loads everything like it never loaded, I set it to recheck for new files every time app opens but I think it should not loose what it already has from the list of movies, shows.

**Comment by courville** [comment]
Ar you by any chance using an external usb drive on a sony tv?
Cf. https://github.com/nova-video-player/aos-AVP/issues/354

**Comment by okan35** [comment]
No sony tv. I use nova on xiaomi mi box ans  my media is on a external hdd that is connected to a laptop which acts as server in the same network with xiaomi.

**Comment by courville** [comment]
OK thus I assume you are using a laptop (which OS?) acting as SMB server: can you confirm this?
Could you please try the latest github pre-release (6.0.34) and provide adb logcat logs of the issue (https://github.com/nova-video-player/aos-AVP/wiki/Debugging)?
Does your network share name changes between two occurrences of the issue?
I might ask you to dump your mediaDb to check what is the problem.

**Comment by okan35** [comment]
Yes, I use a windows 10 laptop with smb.  I will share logs after I try. One thing I noticed that causes issues is if my server is off, which means laptop turned off, nova acts strange so a good server connection check is a must.

I dont think my network had any name changes when this issues happened, as I dont touch my network settings.

I can try to get mediadata db  and share here.

**Comment by courville** [comment]
Nova is designed to keep the full former scraped data when a server is not available and simply tag it hidden (in theory).
To dump your mediaDb you can click 10 times on the nova settings->software decoding, it will enable more options, one of them at the end of the settings list being media database export. You will need to pull it from internal storage located in `/sdcard` on v5 but located in `/sdcard/Android/org.courville.nova/files` on v6.
Please share it with me on a private channel to software at courville dot org.
The ideal would be to get 2 dumps, one when the videos on the network share are not visible and one after your win10 share is available and the full video collection is visible.
